docs/pages/eas-update/getting-started.mdx

```diff
@@ -107,11 +108,68 @@ Then, make any desired changes to your project's JavaScript, styling, or image a
 
 </Step>
 
-## Next step
+<Step label="7">
+## Publish an update
+
+Publishing an update allows:
+
+- Fixing bugs and quickly updating non-native parts of a project instead of creating a new build
+- [Sharing a preview version of an app](/review/overview/) using internal distribution
+
+To publish an update with changes from your project, you can use the `eas update` command, and specify a name for the `branch` and a `message` to describe the update:
+
+<Terminal cmd={['$ eas update --branch [branch-name] --message "[message]"']} />
+
+<Collapsible summary="How does publishing an update work?">
+
+When you publish an update with the `eas update` command, it generates a new update bundle and uploads it to the EAS servers. The `branch` name is used to identify the update and is associated with the uploaded bundle. It is similar to how Git commit works, where every commit is associated with a Git branch.
+
+For example, you want to publish an update for the [`development` build profile](/build/eas-json/#build-profiles). You can use it as the branch name to associate the update with its build profile. When the app requests an update, it will download the update associated with the `development` branch.
```

Comment:
```suggestion
For example, you want to publish an update for the [`development` build profile](/build/eas-json/#build-profiles). You can use it as the channel name to associate the update with its build profile. When the app requests an update, it will download the update associated with the `development` channel.
```